### PR TITLE
Exercise 8

### PR DESCRIPTION
**Pull request by egrfly**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -181,6 +181,53 @@ $ docker compose up cypress --build --abort-on-container-exit
 
 In this case, there is no need to manually start the app - Docker will take care of this for us as well as automatically tearing it down when the tests are complete.
 
+## Deployment with Docker and Azure
+
+If you wish to host your app's container images using the Docker Hub, you will need to [set up a Docker Hub account](https://hub.docker.com/signup).
+Once you have done so, create a public repository called `todo-app`.
+There is also a Docker variable that needs to be set in your `.env` file:
```
Comment (JackMead) [inline README.md:188]: No harm, but is this actually helpful? The `.env` file is currently only loaded in when we run flask, which doesn't really have any interaction with Docker
(Unless docker login actually does read from a `.env` file automatically, but I don't _think_ that's true)

**Review by JackMead — APPROVED** [review]
Nice work, really clear instructions for us in future, and I can see your app is running happily in Azure 😄 Only real suggestion - it would be nice to link that webpage in your Readme!

https://emifly-todo-app.azurewebsites.net/

File: README.md (hunk 1 above)
Comment (emifly) [reply README.md:188]: Docker itself doesn't read from the `.env` file automatically, but it turns out that Docker Compose does! Setting this just helps make sure the tag comes out correctly when running `docker compose build production`

**Comment by emifly** [comment]
Thanks! Link added 😄

**Pull request merged**